main.md

```diff
@@ -591,7 +591,7 @@ trusting/using any of the contents of an SD-JWT:
        1. Ensure that the claim is present as well in `sd_claims` in the SD-JWT.
           If `sd_claims` is structured, the claim MUST be present at the same
           place within the structure.
-       2. Check that the BASE64URL encoded hash of the claim value in the SD-JWT
+       2. Check that the base64url-encoded hash of the claim value in the SD-JWT
```

Comment:
Why did you put ‘-‘ only here? “base64url-encoded” vs “base64url encoded”

Reply:
This hyphen seems to be correct, actually. https://www.grammarly.com/blog/hyphen/
I'll go ahead with the merge.